website/docs/getting-started/setup/upgrade-to-business-edition/kubernetes.md

```diff
@@ -1,166 +1,81 @@
 ---
-description: Follow the guide to upgrade the Appsmith Community Edition installation on Kubernetes to Business Edition.
+description: Follow these steps to upgrade the Appsmith Community Kubernetes installation to the Business Edition.
 ---
 
 # Kubernetes
 
-The Appsmith Business Edition (BE) Helm chart installation supports Horizontal Pod Auto Scaling (HPA), allowing Appsmith pods to scale automatically based on the current load. Additionally, this means that Appsmith pods are managed using a Kubernetes _deployment_ resource instead of a _stateful-set_ resource. Follow this guide to upgrade the Kubernetes installation from Community Edition (CE) to Business Edition (BE).
+This page provides steps to upgrade Appsmith Kubernetes Community Edition to Business Edition.
 
-## Generate license key
+## Prerequisites
 
-Sign up on [customer.appsmith.com](https://customer.appsmith.com/) and generate a trial license key.
+Before upgrading, ensure you have:
 
+- Signed up on [customer.appsmith.com](https://customer.appsmith.com/) and generated a trial license key.
+- At least 2 GB of free storage space for the backup process.
+- Deployed the latest version of Appsmith. Refer to [Update Appsmith](/getting-started/setup/instance-management/update-appsmith#kubernetes) to update the version.
+- Created a backup of your Appsmith instance. For instructions, see [Backup Appsmith Instance](/getting-started/setup/instance-management/appsmithctl?current-command-type=kubernetes-commands#backup-instance).
 
-## Backup data
+## Upgrade to Business Edition
 
-1. Connect to the shell of a running container:
+Follow these steps to upgrade your Appsmith installation:
 
-   ```bash
-   kubectl exec -it <pod> bash
-   ```
+1. Create a folder named `appsmith-ee` on your machine for deployment and data storage. Then, navigate to this folder using the `cd` command.
 
-2. Run the backup command:
+2. Generate the business edition `values.yaml` with:
```

Comment:
can we change the sequence 
like the same steps change the commands in 

2nd point with 3 
3rd point with 4
4th point with 2

i.e 
2- helm repo add appsmith-ee https://helm-ee.appsmith.com
3- helm repo update
4- helm show values appsmith-ee/appsmith > values.yaml